### PR TITLE
Fix faulty indentation of Ecto queries

### PR DESCRIPTION
**Pull request by DanielZendejas**
Hi,

This PR addresses [this issue](https://github.com/elixir-lang/vim-elixir/issues/237) and also cleans up a couple of lines of documentation I found.

This is my first experience ever with Vimscript, so please let me know what you think. Any comment or suggestion is welcome.

### CODE REVIEW

**Comment by pbogut** [comment]
I didn't check it but how it will work if I have variable named `from`? Something like:
```elixir
def smth do
  from = 1
  to = 7
.....
end
```

**Comment by DanielZendejas** [comment]
@pbogut good point, I did not think of that. I'll rewrite this later today so it doesn't indent wrongly.

**Comment by DanielZendejas** [comment]
I fixed the `from` pattern and added a test based on the comment made by @pbogut . I'm not sure why the build failed though, the tests run fine in my local machine. 64 tests failed but I don't think it was because of these changes... did I do something wrong?

Anyway, the code is ready for another review, please let me know what do you think and if I'm missing something.

File: autoload/elixir/indent.vim [hunk 1]
```diff
@@ -14,6 +14,7 @@ let s:MULTILINE_BLOCK = '\%(\<do\>'.s:NO_COLON_AFTER.'\|'.s:MULTILINE_FN.'\)'
 let s:BLOCK_MIDDLE = '\<\%(else\|match\|elsif\|catch\|after\|rescue\)\>'
 let s:BLOCK_END = 'end'
 let s:STARTS_WITH_PIPELINE = '^\s*|>.*$'
+let s:QUERY_FROM = '^\s*from\s*.*\s*in\s*.*,'
```
Comment (kassio) [inline autoload/elixir/indent.vim]: I think we need to improve this regexp, today it matchs with undesired texts too:
nvim

I'd use something like: `^\s*\<from\>.*\<in\>.*,`, wdyt?

Comment (DanielZendejas) [inline autoload/elixir/indent.vim]: You're right, I'll change it later today.

**Comment by DanielZendejas** [comment]
I've replaced the regexp with the one suggested by @kassio and added the example provided as a test.

**Pull request merged**